apps/passport-client/components/screens/AddSubscriptionScreen.tsx

```diff
@@ -204,24 +175,12 @@ export function SubscriptionInfoRow({
     });
   }, [dispatch, subscription]);
 
-  const credentialDisplayOptions = credentialPCD
-    ? pcds.getPackage(credential.type).getDisplayOptions(credentialPCD)
-    : null;
 
   return (
     <InfoRowContainer>
       <FeedName>{info.name}</FeedName>
       <Spacer h={8} />
       <Description>{info.description}</Description>
```

Comment:
We no longer store a fixed credential with the subscription. At some point we are going to need to come back to the question of how we show credentials to the user in the UI, so that they know what's being sent to feed servers.